<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Simulación</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Caudal acequia</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_acequia">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_acequiaPorcent">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Porcentaje: 50%</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_acequiaCaudal">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Caudal: 0.002 m^3/s</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Caudal lluvia</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_lluvia">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_lluviaCaudal">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Caudal: 0.000001 m^3/s</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pause">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_reset">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_1x">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>1x</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10x">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>10x</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_100x">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>100x</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_250x">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>250x</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Válvula</string>
      </property>
      <widget class="QPushButton" name="pushButton_abrir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Abrir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_cerrar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_valvula">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Válvula cerrada</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tiempo simulado: 0 segundos.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Configuración</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>321</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Alberca</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nivel máximo:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nivel inicial:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_albercaInit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_albercaMax">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
       <property name="value">
        <double>240.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Área de la base:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_albercaArea">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>100</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>m^2</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>321</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Acequia</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nivel inicial:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nivel máximo:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_acequiaInit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_acequiaMax">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>m^3/s</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>m^3/s</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>321</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Lluvia</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Caudal inicial:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_lluviaInit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>m^3/s</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Válvula de salida</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Radio desagüe:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_valvulaRadio">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>62</width>
         <height>26</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Base de datos</string>
      </property>
      <widget class="QLabel" name="labelNombrebase">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nombre">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Guardar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Cargar">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_nivelTotal">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel: 0 cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_caudalsalida">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>500</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Caudal de salida: 0 m^3/s</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>420</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelVsalida">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelQacequia">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>580</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelVentrada">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>610</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
